pandas/plotting/_core.py

```diff
@@ -2772,20 +2772,46 @@ def box(self, by=None, **kwds):
 
     def hist(self, by=None, bins=10, **kwds):
         """
-        Histogram
+        Draw one histogram of the DataFrame's Series using matplotlib.
+
+        A histogram is a representation of the distribution of data.
+        This function groups the values of all given Series in the DataFrame
+        into bins, and draws all bins in only one matplotlib.AxesSubplot. This
+        is useful when the DataFrame's Series are in a similar scale.
 
         Parameters
         ----------
-        by : string or sequence
+        by : str
```

Comment:
Why removing sequence? A sequence of string can be passed to select the columns to be plotted.